### PR TITLE
Uploading all files

### PR DESCRIPTION
**Pull request by pcwebb**
Assignment warmup #03

### CODE REVIEW

**Comment by cheuschober** [comment]
retest this please

**Comment by pcwebb** [comment]
Can't figure out how to re-test.

**Comment by cheuschober** [comment]
@pcwebb,

If it wasn't immediately apparent to you, that's a request to jenkins to have him retest your work. You or I can put that in a comment which jenkins will read and then execute the tests again. I had to ask him to retest everyone's work since it appears he went on a holiday on Thursday along with the rest of the school.

**Comment by pcwebb** [comment]
@cheuschober,
It was not immediately apparent. I knew about Thursday but I thought the comment was a request for me to re-test and I was trying to figure out how. Sometimes this is all very confusing. Thanks for getting back to me. Next time I'll asks sooner.

**Comment by cheuschober** [comment]
@pcwebb, no worries, it looks like your work was stellar either way.

File: task_11.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Warmup exercise Task_11"""
+
+ESCAPE_STRING = '''\\n'"'''
```
Comment (cheuschober) [inline task_11.py]: This works but you should also learn how to simply escape these strings. See this week's QA review on how to escape strings.

File: task_12.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Warmup exercise Task_12."""
+
+import decimal
+import fractions
+
+INTVAL = 1
+FLOATVAL = 0.1
+DECVAL = decimal.Decimal('0.1')
```
Comment (cheuschober) [inline task_12.py]: Excellent job remembering to use the string form of the decimal here!

**Pull request merged**